cancel-in-progress/parent_workflow.go

```diff
@@ -0,0 +1,109 @@
+package cancel_in_progress
+
+import (
+	"errors"
+	"go.temporal.io/sdk/log"
+	"go.temporal.io/sdk/workflow"
+)
+
+const ParentWorkflowSignalName = "parent-workflow-signal"
```

Comment:
IMO the signal name should reference what it does rather than who it's to, e.g. `const RestartChildSignalName = "restart-child-signal"`